windows/src/wmi.cc

```diff
@@ -25,9 +27,9 @@ namespace leatherman { namespace windows {
     {
 #ifdef LEATHERMAN_I18N
         // LOCALE: format a pointer as hex for printing an error message.
-        return lth_locale::format("{1} (0x{2,num=hex})", s, hres);
+        return _("{1} (0x{2,num=hex})", s, hres);
 #else
-        return lth_locale::format("%1% (0x%2$#x)", s, hres);
+        return _("%1% (0x%2$#x)", s, hres);
```

Comment:
boost::format and boost::locale::format have different special formats, that's why it does `%2$#x` and `{2,num=hex}`. I'm not aware of a way to do this in a compatible way, and handling all the special cases to convert would be crazy. So I think this special case is still necessary.